### DISCUSSION TITLE
Snackbar

### DISCUSSION THREAD

**Discussion by RonPeters** [microsoft/fluentui-blazor#378] [Ideas]
I was surprised to not find any Snackbar functionality. Most component toolkits these days include it. Are there any plans for a Snackbar?

**Comment by vnbaaij** [comment]
Yes, we plan to have a Toast component in the next version.
Reason it was not there before is we only offered wrappers for the Fluent UI Web Components. For V3 we're relaxing that restriction and are adding new components. See https://baaijte.net/blog/microsoft-fluentui-blazor-3.0/
